src/apps/competitions/models.py

```diff
@@ -67,6 +173,7 @@ class Phase(models.Model):
     name = models.CharField(max_length=256)
     description = models.TextField(null=True, blank=True)
     execution_time_limit = models.PositiveIntegerField(default=60 * 10)
+    auto_migration = models.BooleanField(default=False)
```

Comment:
We may want to rename this to be a little more clear, are we auto migrating _from_ this phase or _to_ this phase. I image from, but clarity in naming could be helpful, especially in potentially creating competitions where they only want to migrate certain phases of the comp.

Also, the competition unpacking task should be updated to look for auto migration in phases on the yaml.

Reply:
See above, variable name is clearer now: `auto_migrate_to_this_phase`